cas-server-webapp-support/src/main/java/org/jasig/cas/web/support/CookieRetrievingCookieGenerator.java

```diff
@@ -103,9 +103,14 @@ public void addCookie(final HttpServletRequest request, final HttpServletRespons
      * @return the cookie value
      */
     public String retrieveCookieValue(final HttpServletRequest request) {
-        final Cookie cookie = org.springframework.web.util.WebUtils.getCookie(
-                request, getCookieName());
-        return cookie == null ? null : this.casCookieValueManager.obtainCookieValue(cookie, request);
+        try {

```

Comment:
Good point, I'll yank this out into another PR. 

The fix that was applied is not enough, because if `obtainCookieValue` throws an error, CAS would still crash.  (say, IP mismatch)